web/webpack.config.js

```diff
@@ -113,13 +115,15 @@ if (!isProduction) {
 
 let plugins = [
   new WriteFilePlugin(),
-  new CopyWebpackPlugin(copyWebpackCommands),
+  new CopyWebpackPlugin({ patterns: copyWebpackCommands }),
   new DefinePlugin({
     IS_WEB: JSON.stringify(true),
     'process.env.SDK_API_URL': JSON.stringify(process.env.SDK_API_URL || LBRY_WEB_API),
   }),
   new ProvidePlugin({
-    __: ['i18n.js', '__'],
+    Buffer: ['buffer', 'Buffer'],
+    process: 'process/browser',
+    __: [path.resolve(path.join(__dirname, '../ui/i18n')), '__'],
```

Comment:
Code paths that make user of these fallbacks are of particular interest to test. Webpack no longer automatically rolls these in, so they need to be manually wired in.